requirements-dev.txt

```diff
@@ -33,6 +33,7 @@ mypy>=0.982
 types-beautifulsoup4
 types-freezegun
 types-jsonschema
+types-mock
```

Comment:
we should not be using `mock` -- we should be using `unittest.mock`

Reply:
I've been sloooowly trying to kill it, some crappy third party dependency pulls it in still though 😭

Reply:
```
$ grep -i '^requires-dist.*mock[^;]*$' .venv/lib/python3.8/site-packages/*/METADATA
.venv/lib/python3.8/site-packages/exam-0.10.6.dist-info/METADATA:Requires-Dist: mock
```

bleh

Reply:
Lol, `exam` was built at Disqus and I think the guy who wrote/maintains it now mostly works in rubby. So we'll have to fork it ourselves if we want to fix it.

Reply:
yeahhhh I've just been slowly factoring it out instead